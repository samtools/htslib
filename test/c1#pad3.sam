@SQ	SN:c1	LN:16
@RG	ID:p.sam	SM:unknown	LB:p.sam
s0a	0	c1	6	0	5I6P5M	*	0	0	AACCGCGGTT	*	RG:Z:p.sam
s0b	0	c1	6	0	5I6P5M	*	0	0	AACCGCGGTT	*	RG:Z:p.sam
s0c	0	c1	6	0	5I6P5M	*	0	0	AACCGCGGTT	*	RG:Z:p.sam
s0d	0	c1	6	0	5I6P5M	*	0	0	AACCGCGGTT	*	RG:Z:p.sam
s1	0	c1	6	0	11I5M	*	0	0	AACCGGTTAACCGGTT	*	RG:Z:p.sam
s2	0	c1	6	0	5I1P4I1P5M	*	0	0	AACCGTTAACGGTT	*	RG:Z:p.sam
s3	0	c1	6	0	8I3P5M	*	0	0	AACCGGTTCGGTT	*	RG:Z:p.sam
s4	0	c1	6	0	5I3P3I5M	*	0	0	AACCGAACCGGTT	*	RG:Z:p.sam
s5	0	c1	6	0	4I3P2I2P1D4M	*	0	0	AACCTAGGTT	*	RG:Z:p.sam
s6	0	c1	6	0	2I3P6I3D2M	*	0	0	AAGTTAACTT	*	RG:Z:p.sam
s7	0	c1	6	0	4I7P1D4M	*	0	0	AACCGGTT	*	RG:Z:p.sam
s8	0	c1	6	0	7P2I2P	*	0	0	TA	!!	RG:Z:p.sam
