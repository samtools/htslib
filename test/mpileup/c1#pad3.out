c1	6	11	^!C^!C^!C^!C^!C^!C^!C^!C^!*^!*^!*	~~~~~~~~~~~
c1	7	11	GGGGGGGGG*G	~~~~~~~~~~~
c1	8	11	GGGGGGGGG*G	~~~~~~~~~~~
c1	9	11	TTTTTTTTTTT	~~~~~~~~~~~
c1	10	11	T$T$T$T$T$T$T$T$T$T$T$	~~~~~~~~~~~
